docs/content/en/docs/release-notes/_index.md

```diff
@@ -0,0 +1,123 @@
+
+---
+title: "K8ssandra release notes"
+linkTitle: "Release notes"
+weight: 1
+description: Release notes for the open-source K8ssandra community project.
+---
+
+K8ssandra provides a production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. 
+
+Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API. 
+
+{{% alert title="Note" color="success" %}}
+The **K8ssandra 1.1.0** release implements a number of changes, enhancements, and bug fixes. This Release Notes topic lists a subset of the key updates. For the complete list, see the [CHANGELOG](https://github.com/k8ssandra/k8ssandra/blob/main/CHANGELOG.md).  
+{{% /alert %}}
+
+**Release date:** 09-April-2021.
+
+## Prerequisites
+
+* A Kubernetes v1.16 or later environment - local or via a supported cloud provider
+* [Helm](https://helm.sh/) v3
+
+## Supported Kubernetes environments
+
+* Open-source [kubernetes.io](https://kubernetes.io)
+* [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE)
+* [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/) (AKS)
+* [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS)
+* [MiniKube](https://minikube.sigs.k8s.io/docs/)
+* [Kind](https://kind.sigs.k8s.io/)
+* [K3D](https://k3d.io/)
+
+## K8ssandra deployed components
+
+The K8ssandra helm chart deploys the following components. Some are optional, and depending on the configuration, may not be deployed:
+
+* Apache Cassandra - the deployed version depends on the configured `serverImage` setting:
```

Comment:
There is no `serverImage` property in the k8ssandra chart. The user just needs to set the `cassandra.version` property.

Reply:
Done